### PR TITLE
feat: atmosphere pro contributing tutorial

### LINKED ISSUE #12: feat: atmospherePro codelab
lead: TBD
- [ ] Discussion on what parts of the app we want to highlight in the codelab
- [ ] Write out each codelab step

### PR DESCRIPTION
**Pull request by JeremyTubongbanua**
I made a quick tutorial for developers who want to contribute to atmosphere pro

closes #12

### CODE REVIEW

**Comment by xavierchanth** [comment]
The atmospherePro content was supposed to be on how the app works, but this is good content that we shouldn't waste. Maybe we can discuss early next week where we want this content to go.

**Comment by xavierchanth** [comment]
The atmospherePro content was supposed to be on how the app works, but this is good content that we shouldn't waste. Maybe we can discuss early next week where we want this content to go.

**Comment by xavierchanth** [comment]
The atmospherePro content was supposed to be on how the app works, but this is good content that we shouldn't waste. Maybe we can discuss early next week where we want this content to go.

**Comment by xavierchanth** [comment]
Maybe the best place for it is under contribute, and we move the current contribute content under a "this site" folder.

**Comment by JeremyTubongbanua** [comment]
Sounds good

**Comment by JeremyTubongbanua** [comment]
I will find a better place to put this, converting it to draft

File: content/docs/contribute/atmosphere-pro/1-introduction/index.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+order: 1
+---
+
+In this tutorial, we will be cloning our project so that you can make contributions to our end-to-end encrypted file transfer app, atmosphere pro. To get started, you will need the following:
+
+1. [Git](https://git-scm.com/downloads)
+2. A [GitHub](https://github.com/) account
+3. A code editor like [VSCode](https://code.visualstudio.com/download) for coding in Dart/Flutter.
+4. The [Flutter](https://docs.flutter.dev/get-started/install) framework
+5. An environment to test and run the application ([Android](https://developer.android.com/studio?gclid=CjwKCAjwx7GYBhB7EiwA0d8oe3wbfdyIU9rVXgxc6VwabUpnLdjnhEZHKxGupUw97lF91AZkbbNx9BoCB-cQAvD_BwE&gclsrc=aw.ds), [iOS](https://developer.apple.com/xcode/), [macOS](https://docs.flutter.dev/get-started/install/macos#macos-setup), or [Windows](https://docs.flutter.dev/get-started/install/windows)).
```
Comment (xavierchanth) [inline content/docs/contribute/atmosphere-pro/1-introduction/index.md]: Please remove the parameters from the Android Studio link

File: content_dev/docs/tutorials/atmosphere-pro/_index.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 layout: codelab-list # The layout for a codelab list (think of this as a title page for the code lab)
 
 title: "atmosphere pro" # Title of the codelab
-lead: Learn how the atmosphere pro app works # Description of the codelab
+lead: Make Contributions to our P2P File Transferring App # Description of the codelab
```
Comment (xavierchanth) [inline content_dev/docs/tutorials/atmosphere-pro/_index.md]: Revert this change please

**Review by xavierchanth — CHANGES_REQUESTED** [review]
A couple of small changes noted, but otherwise LGTM

File: content/docs/contribute/atmosphere-pro/1-introduction/index.md (hunk 1 above)
Comment (JeremyTubongbanua) [reply content/docs/contribute/atmosphere-pro/1-introduction/index.md]: Done in https://github.com/atsign-foundation/docs.atsign.com/pull/124/commits/d92f9a1ad8991954d0a5c2503e13d4baa01a1b38

File: content_dev/docs/tutorials/atmosphere-pro/_index.md (hunk 2 above)
Comment (JeremyTubongbanua) [reply content_dev/docs/tutorials/atmosphere-pro/_index.md]: Hopefully this is it - https://github.com/atsign-foundation/docs.atsign.com/pull/124/commits/5c62ffe18646aca6ee1aff897d834d03ba153ee5

**Review by xavierchanth — APPROVED** [review]
